scripts/make.py

```diff
@@ -62,14 +62,14 @@ def edit_linker(offset):
 	file_change_line("linker.ld", 4, "\t\trom     : ORIGIN = (0x08000000 + " + hex(offset) + "), LENGTH = 32M\n")
 	
 def edit_insert(offset):
-	file_change_line("./scripts/insert.py", 11, "OFFSET_TO_PUT = " + hex(offset) + '\n')
-	file_change_line("./scripts/insert.py", 12, 'SOURCE_ROM = "' + ROM_NAME + '"\n')
+	file_change_line("./scripts/insert.py", 12, "OFFSET_TO_PUT = " + hex(offset) + '\n')
+	file_change_line("./scripts/insert.py", 13, 'SOURCE_ROM = "' + ROM_NAME + '"\n')
 		
 def build_code():
-	os.system("python scripts/build.py")
+	os.system("python3 scripts/build.py")
```

Comment:
my OS has both python2 and python3 so we need this to force people to use python3

Reply:
This one is updated to handle both `python3` and `python` for convenience